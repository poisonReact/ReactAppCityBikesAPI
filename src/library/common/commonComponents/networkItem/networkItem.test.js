import React from 'react'
import renderer from 'react-test-renderer'
import { mount, shallow } from 'enzyme'
import NetworkItem from 'library/common/commonComponents/networkItem/networkItem'

it('render correctly NetworkItem component', () => {
    const NetworkItemComponent = renderer.create(<NetworkItem />).toJSON()
    expect(NetworkItemComponent).toMatchSnapshot()
})
/*
it("check prop network by default", () => {
    const NetworkItemComponent = mount(<NetworkItem />);
    expect(NetworkItemComponent.find("p").text()).toEqual("No Data");
});
*/
it('should render a document title', () => {
    const wrapper = shallow(
        <NetworkItem />
    );
    expect(wrapper.prop('network')).toEqual({company: ['No Data'], id: 'No Data'});
});