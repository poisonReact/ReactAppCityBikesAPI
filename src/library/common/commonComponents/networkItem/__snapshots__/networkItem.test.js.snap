// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly NetworkItem component 1`] = `
<div
  className="networkItemWrapper"
>
  <div
    onClick={[Function]}
  >
    <div
      className="networkItemName"
    >
      No Data
    </div>
    <div
      className="networkItemId"
    >
      (No Data)
    </div>
  </div>
</div>
`;
